molgenis/molgenis-emx2
# feat(central): remove and enable/disable users via admin ui

fixes: #3981

What are the main changes you did:
- remove user button
- enable and disable user accounts toggle

how to test:
- Go to: apps/central/#/admin/users
- create a user / remove users 
- try to disable an user and login

todo:
- [x]  combine removeUser and deleteUser?
- [x] backend and frontend should not be able to disable "anonymous and admin"
